pages/chapter4.tsx

```diff
@@ -44,6 +46,18 @@ export default function Chapter3() {
                   details={devOscar[1].details}
                   code={() => <ReversingAnArray />}
                 />
+                <PageCard
+                  id={devOscar[2].id}
+                  title={devOscar[2].title}
+                  details={devOscar[2].details}
+                  code={() => <Alist />}
```

Comment:
All the List code should be removed from this PR, including the Alist components